python/ql/test/experimental/meta/InlineTaintTest.qll

```diff
@@ -30,24 +30,34 @@ DataFlow::Node shouldNotBeTainted() {
   )
 }
 
-class TestTaintTrackingConfiguration extends TaintTracking::Configuration {
-  TestTaintTrackingConfiguration() { this = "TestTaintTrackingConfiguration" }
+module Conf {
```

Comment:
Would like a comment explaining _why_ we have this module, such as

```suggestion
// this module allows the configuration to be imported in other `.ql` files without the
// top level query predicates of this file coming into scope.
module Conf {
```

Reply:
Good point, the PR message is not really handy when reading the code...